maintainers/maintainer-list.nix

```diff
@@ -3487,6 +3487,12 @@
     email = "t@larkery.com";
     name = "Tom Hinton";
   };
+  hjones2199 = {
+    email = "hjones2199@gmail.com";
+    github = "hjones2199";
+    githubId = 5525217;
+    name = "Hunter Jones";
+  };
```

Comment:
Please put this into a separate commit with the message ``maintainers: add hjones2199``.

Reply:
Just rebased onto master, since I added myself to the maintainer list in a different pull request this issue is solved. Working on your other suggestions as well.